chevah/empirical
# [#2380] Refactor the code for EXCEPTED_DELAYED_CALLS.

Scope
=====

In some cases we want to have delayed calls for which the reactor run should not wait to be finalized...for example when we have a looping call.

Changes
=======

Clean the code handling EXCEPTED_DELAYED_CALLS


How to test
========

reviewers: @alibotean 

check that changes make sense

this is released as the py3 compatible version... but for server we need a backport which will be release from a separate branch https://github.com/chevah/empirical/tree/py2-backports

Thanks!
